examples/python/ros/rerun_urdf.py

```diff
@@ -0,0 +1,83 @@
+import io
+import os
+from typing import Optional, cast
+from urllib.parse import urlparse
+
+import numpy as np
+import rerun as rr
+import trimesh
+from ament_index_python.packages import get_package_share_directory
+from std_msgs.msg import String
+from yourdfpy import URDF
+
+
+def ament_locate_package(fname: str) -> str:
+    """Helper to locate urdf resources via ament."""
+    if not fname.startswith("package://"):
+        return fname
+    parsed = urlparse(fname)
+    return os.path.join(get_package_share_directory(parsed.netloc), parsed.path[1:])
+
+
+def load_urdf_from_msg(msg: String) -> URDF:
+    """Load a URDF file using `yourdfpy` and find resources via ament."""
+    f = io.StringIO(msg.data)
+    return URDF.load(f, filename_handler=ament_locate_package)
+
+
+def log_scene(scene: trimesh.Scene, node: str, path: Optional[str] = None, timeless: bool = False) -> None:
```

Comment:
This feels like a useful function to have even outside the ROS example. Perhaps worth upgrading it to the SDK (in a separate PR)?

Reply:
Agreed. https://github.com/rerun-io/rerun/issues/1536